packages/vuetify/src/components/VCalendar/util/props.ts

```diff
@@ -123,9 +123,12 @@ export default {
   },
   category: {
     categories: {
-      type: [Array, String],
+      type: [Array, String] as PropType<CalendarCategory[] | string>,
```

Comment:
is this correct? can categories prop value be a singular string?

Reply:
Yes, the string can be a list of categories comma separated. That's the original design.